### PR TITLE
SLM-303. Add breadcrumbs to allow user to go back to home page.

### PR DESCRIPTION
**Pull request by dhirajshettymoj**

### CODE REVIEW

File: server/middleware/populateCurrentUser.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function populateCurrentUser(userService: UserService): RequestHa
         const user = res.locals.user && (await userService.getUser(res.locals.user.token))
         if (user) {
           res.locals.user = { ...user, ...res.locals.user }
+          res.locals.isPrisonUser = res.locals.user.authSource === 'nomis'
```
Comment (tpmcgowan) [inline server/middleware/populateCurrentUser.ts:17]: Is this needed? Can't see `isPrisonUser` referenced anywhere?

File: server/middleware/breadcrumbs.ts
Comment (tpmcgowan) [inline server/middleware/breadcrumbs.ts:1]: Looks like these breadcrumbs are always hard-coded to the same value? If so, perhaps a bit overkill to have a whole middleware for it - could just have the defaults in `server/views/partials/breadcrumbs.njk` (i.e. the `else` block in that file)?

Obviously works fine though and this is extensible if needed so up to you 🤷‍♂️

**Review by tpmcgowan — COMMENTED** [review]
Looks fine - just comments about a possibly unneeded line and (optionally) thinking about if middleware needed if they're always the same values

File: server/middleware/populateCurrentUser.ts (hunk 1 above)
Comment (dhirajshettymoj) [reply server/middleware/populateCurrentUser.ts]: Updated.

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**